pkg/controller/get-or-create-drift-issue/main.go

```diff
@@ -85,15 +91,12 @@ func (ctrl *Controller) createIssue(ctx context.Context, logE *logrus.Entry, rep
 	}
 	logE.Info("created an issue")
 
-	githubactions.SetOutput("issue_number", strconv.Itoa(issue.GetNumber()))
-	githubactions.SetOutput("issue_state", "open")
+	ctrl.action.SetEnv("TFACTION_DRIFT_ISSUE_NUMBER", strconv.Itoa(issue.GetNumber()))
+	ctrl.action.SetEnv("TFACTION_DRIFT_ISSUE_STATE", "open")
 
-	githubactions.SetEnv("TFACTION_DRIFT_ISSUE_NUMBER", strconv.Itoa(issue.GetNumber()))
-	githubactions.SetEnv("TFACTION_DRIFT_ISSUE_STATE", "open")
-
-	issueURL := fmt.Sprintf("%s/%s/%s/pull/%v", param.GitHubServerURL, repoOwner, repoName, issue.Number)
```

Comment:
Fix a bug.

Change `issue.Number` to `*issue.Number`. `issue.Number` is `*int`